incubator/kube-downscaler/values.yaml

```diff
@@ -0,0 +1,47 @@
+# Default values for kube-downscaler.
+# Declare variables to be passed into your templates.
+
+# How many kube-downscaler pods should run in deployment
+replicaCount: 1
+name: kube-downscaler
+debug:
+  enable: 'False'
+
+# Namespace where kube-downscaler is going to be deployed.
+namespace:
+  # Deployment will query all namespaces if left empty:
+  active_in: ''
+
+# Default is 1 minute.
+interval: 60
+
+rbac:
+  # If true, create & use RBAC resources
+  create: true
+  # Ignored if rbac.create is true
+  serviceAccountName: default
+
+image:
+  repository: hjacobs/kube-downscaler
+  tag: 0.6
+  pullPolicy: IfNotPresent
+
+resources:
+  limits: {}
+  requests: {}
+  # limits:
+  #   cpu: 50m
+  #   memory: 200Mi
+  # requests:
+  #   cpu: 50m
+  #   memory: 200Mi
+
+  tolerations: []
+
+  nodeSelector: {}
+
+  affinity: {}
```

Comment:
indentation looks off for `affinity`, `nodeSelector` and `tolerations`

Reply:
Good catch!